pkg/util/ctxutil/ctx.go

```diff
@@ -0,0 +1 @@
+package ctxutil
```

Comment:
Why do we need this empty package?

Reply:
I put `WithContext` in here, but then I realized it needed to be grouped with logging functions. I must have forgotten to remove it.I'll remove this empty package.